runtime-modules/content/src/benchmarks/benchmarking.rs

```diff
@@ -0,0 +1,226 @@
+#![cfg(feature = "runtime-benchmarks")]
+
+use crate::permissions::*;
+use crate::types::ChannelOwner;
+use crate::Module as Pallet;
+use crate::{Call, ChannelById, Config, Event};
+use frame_benchmarking::benchmarks;
+use frame_support::storage::StorageMap;
+use frame_support::traits::Get;
+use frame_system::RawOrigin;
+use sp_arithmetic::traits::{One, Saturating};
+use sp_runtime::SaturatedConversion;
+
+use super::{
+    assert_last_event, generate_channel_creation_params, insert_content_leader, insert_curator,
+    insert_distribution_leader, insert_storage_leader, member_funded_account,
+    setup_worst_case_curator_group_with_curators, worst_case_channel_agent_permissions,
+    worst_case_content_moderation_actions_set, CreateAccountId, RuntimeConfig, CURATOR_IDS,
+    DEFAULT_MEMBER_ID,
+};
+
+benchmarks! {
+    where_clause {
+        where
+            T: RuntimeConfig,
```

Comment:
We should use Config, is set this way in the rest of the project
```suggestion
            T: Config,
```

Reply:
This is just a simplification, so instead of writing:
```rust
              T: balances::Config,
              T: membership::Config,
              T: storage::Config,
              T: working_group::Config<StorageWorkingGroupInstance>,
              T: working_group::Config<DistributionWorkingGroupInstance>,
              T: working_group::Config<ContentWorkingGroupInstance>,
              T: Config 
```

We can just use this `T: RuntimeConfig` "alias trait" (in which we define all the traits that are needed).
It seems to work just fine, do you see any issues with this approach?